components/WebServer/routes/api/media/media.js

```diff
@@ -11,15 +13,25 @@ module.exports = (webserver) => {
             try {
                 const conversation = await conversationModel.getConvoById(req.params.conversationId)
                 if (conversation.length === 1 && conversation[0].metadata && conversation[0].metadata.audio && conversation[0].metadata.audio.filepath) {
+                  if(req.headers.accept.indexOf('audio/mpeg') >= 0) {
```

Comment:
Feel like the if above need to be split depend on the headers.accept request 
(conversation[0].metadata.audio)